### ISSUE TITLE
Docker Image Format v1 and Docker Image manifest version 2

### ISSUE THREAD

**Issue by hiroyuki-sato** [CenterForOpenScience/fakecas#17]
Hello, team. Is it possible to create an image using V2 format?

https://docs.docker.com/engine/deprecated/#pushing-and-pulling-with-image-manifest-v2-schema-1

`docker pull  quay.io/centerforopenscience/fakecas:master`

```
[DEPRECATION NOTICE] Docker Image Format v1 and Docker Image manifest version 2, schema 1 support is disabled by default and will be removed in an upcoming release. Suggest the author of quay.io/centerforopenscience/fakecas:master to upgrade the image to the OCI Format or Docker Image manifest v2, schema 2. More information at https://docs.docker.com/go/deprecated-image-specs/
```

> An environment variable (`DOCKER_ENABLE_DEPRECATED_PULL_SCHEMA_1_IMAGE`) is added in Docker v26.0 that allows re-enabling support for these image formats in the daemon. This environment variable must be set to a non-empty value in the daemon's environment (for example, through a [systemd override file](https://docs.docker.com/config/daemon/systemd/)). Support for the `DOCKER_ENABLE_DEPRECATED_PULL_SCHEMA_1_IMAGE` environment variable will be removed in Docker v27.0 after which this functionality is removed permanently.

I downgraded `docker` to 25.0.4 as a workaround.

## Environment

* Docker version 27.0.3, build 7d4bcd8
* OS: Ubuntu 22.04

**Comment by mfraezz** [comment]
Hi @hiroyuki-sato,

Thanks for bringing this to our attention.

We merged PR #16 for docker schema changes two months ago, but Quay was unable to successfully build it.
I've built the image locally and pushed it up with the tags `develop`, `master`, and `24.0.0`. You should be able to pull a newer image now, sorry for the wait.

The old image is available at `19.0.1`, if necessary.

**Comment by hiroyuki-sato** [comment]
@mfraezz Thanks. I've missed #16.
